### PR TITLE
[DFFA-7] Style the Product Details Page using CSS

### PR DESCRIPTION
**Pull request by Seyi01**
Screenshot (34)
Screenshot (35)
Screenshot (36)
Screenshot (37)
Screenshot (38)
Screenshot (39)
Screenshot (40)
Screenshot (33)

### CODE REVIEW

**Comment by Ireti203** [comment]
Weldone @Seyi01. Your work is not responsive enough. Also, attach the screenshot of the laptop full screensize without showing the console.

**Comment by Seyi01** [comment]
Screenshot (52)
Screenshot (53)
Screenshot (49)
Screenshot (50)
Screenshot (51)
Screenshot (49)
Screenshot (50)
Screenshot (51)
Screenshot (53)


@Ireti203 . Good afternoon ma'am. Yet to get your feedback on DFFA-6 too. thank you for always.

File: submissions/seyifunmiOjo/css/css2/product.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/seyifunmiOjo/css/css2/product.html:1]: You don't have to create a new HTML file for this task. Kindly update the product page you have in your HTML folder. It should be a continuation

**Review by adaugochi — COMMENTED** [review]
Well done

All images should be inside the `images` folder that is in your `html` folder.

File: submissions/seyifunmiOjo/css/css2/product.html (hunk 1 above)
Comment (Seyi01) [reply submissions/seyifunmiOjo/css/css2/product.html]: Done ma. I've put it in its right folder @adaugochi

**Comment by Seyi01** [comment]
I've corrected this ma. Please check and review ma.

**Review by adaugochi — APPROVED** [review]

**Pull request merged**